metadata2beaker.rb

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env ruby
+require 'json'
+
+def metadata2beaker(module_path, pidfile: false, hostname: true)
+  metadata = JSON.parse(File.read(File.join(module_path, 'metadata.json')))
+
+  metadata['operatingsystem_support'].each do |os|
+    if ['CentOS', 'Fedora', 'Debian', 'Ubuntu'].include?(os['operatingsystem'])
+      os['operatingsystemrelease'].each do |release|
+        name = "#{os['operatingsystem'].downcase}#{release.tr('.', '')}-64"
+
+        options = {}
+        options[:hostname] = "#{name}.example.com" if hostname
+        # Docker messes up cgroups and modern systemd can't deal with that when
+        # PIDFile is used.
+        if pidfile
+          case os['operatingsystem']
+          when 'CentOS'
+            options[:image] = 'centos:7.6.1810' if release == '7'
+          when 'Ubuntu'
```

Comment:
@alexjfisher I recall you also had to apply this for some Ubuntu version but I couldn't find which version. Do you recall?

Reply:
https://github.com/voxpupuli/puppet-zabbix/pull/673